.github/maintainers_guide.md

```diff
@@ -108,33 +108,47 @@ If you want to test the package locally you can.
 
 #### test.pypi.org deployment
 
-[TestPyPI](https://test.pypi.org/) is a separate instance of the
-Python Package Index that allows you to try distribution tools and
-processes without affecting the real index. This is useful with changes that
-relate to the package itself, example the contents of the `pyproject.toml`
+[TestPyPI](https://test.pypi.org/) is a separate instance of the Python Package
+Index that allows you to try distribution tools and processes without affecting
+the real index. This is useful with changes that relate to the package itself,
+example the contents of the `pyproject.toml`
 
 The following can be used to deploy this project on <https://test.pypi.org/>.
 
 ```zsh
-# Set the new version with SLACK_CLI_HOOKS_VERSION
-SLACK_CLI_HOOKS_VERSION="1.2.3" ./scripts/deploy_to_test_pypi.sh
+
+./scripts/deploy_to_test_pypi.sh
 ```
 
 #### Development Deployment
 
-Releases for this library are automatically generated off of git releases. Before
-creating a new release, ensure that everything on a stable branch has landed, then
-[run the tests](#run-all-the-unit-tests).
-
-1. Create a new GitHub Release from the
+Releases for this library are automatically generated off of git releases.
```

Comment:
`git` has no concept of releases, only tags. Did you mean GitHub Releases here?

Reply:
@filmaj good catch 💯  currently the [Github Action that deploys a new version](https://github.com/slackapi/python-slack-hooks/blob/main/.github/workflows/release.yml) of the project to Pypi is triggered by publishing a new Github Release, the instructions ask to create a tag along with the GitHub release
Should this action be triggered by the tag creation instead?

Reply:
I think Release creation is a nice underlying trigger, as it forces the maintainer to create a GitHub Release (which in turn will create a tag).